native/exe/SageLauncher/launcher.cpp

```diff
@@ -1339,8 +1339,8 @@ int launchJVMSage(LPSTR lpszCmdLine, HWND hWnd, BOOL bClient, BOOL bService)
 
 	char winBuf[16];
     char stdOutBuf[16];
-    sprintf(winBuf, "%d", hWnd);
-    sprintf(stdOutBuf, "%d", stdOutHandle);
+    sprintf(winBuf, "0x%p", hWnd);

```

Comment:
These can't be hex; they need to be decimal because they are parsed as base 10 integers from Java.